server/scratchy_server/resources/userres.py

```diff
@@ -24,14 +24,21 @@ def delete(self, userId):
         return None
 
 
-class NoIdUserRes(MethodResource):
+class AllUserRes(MethodResource):
 
-    @doc(tags=['User'])
-    @marshal_with(UserSchema)
+    @doc(tags=['AllUser'])
```

Comment:
Actually there is no need to use a new tag, it's relating to the user resource it should be visible in the user section of the swagger.

I've tested, replace `AllUser` by `User` it works like a charm and it's easier to read it in the swagger UI also :
![UserTag](https://user-images.githubusercontent.com/1092728/119275291-76986a00-bc14-11eb-8922-f1a1eb49777b.png)